### PR TITLE
Archive 2021 ideas list

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
This PR makes it explicit that we were not selected for 2021 as an organization. I considered removing the page completely, but I think it is valuable to keep it there for historical reasons. If you disagree, please tell me so!

The structure change I did was to include everything that was there as a sub-section of the now top section: 2021. We'll add a new top section next year.

#### Ticket Link
--

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]

**Review by emilyacook — APPROVED** [review]

File: site/content/contribute/gsoc/_index.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 ---
-title: "Google Summer of Code 2021"
-description: "Ideas list for Google Summer of Code 2021"
+title: "Google Summer of Code"
+description: "Ideas list for Google Summer of Code"
 section: "contribute"
 ---
 
-We are happy to announce that Mattermost is applying to [Google Summer of Code 2021](https://summerofcode.withgoogle.com) as an organization!
+# 2021 ideas
```
Comment (justinegeffen) [inline site/content/contribute/gsoc/_index.md:7]:
```suggestion
# GSoC 2021 Ideas
```

File: site/content/contribute/gsoc/_index.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+
+**Note**: Mattermost was **not** selected as an organization for Google Summer of Code 2021. All that follows is kept here for historical reasons. Hopefully, we will be able to use some of the ideas in 2022.
```
Comment (justinegeffen) [inline site/content/contribute/gsoc/_index.md:9]:
```suggestion
**Note**: Mattermost was **not** selected as an organization for Google Summer of Code 2021. All that follows is kept here for historical reasons. Hopefully, we'll be able to use some of the ideas in 2022.
```

File: site/content/contribute/gsoc/_index.md [hunk 3]
```diff
@@ -38,13 +40,13 @@ Here’s a sneak peek of what’s coming next:
 
 Make sure to take a look at [the full Google Summer of Code timeline](https://summerofcode.withgoogle.com/how-it-works/#timeline) for more details!
 
-# Ideas list
+## Ideas list
 
-## Core features
+### Core features
 
 All ideas in the _Core features_ category propose direct changes to the Mattermost product, either within the server, or within the web client.
```
Comment (justinegeffen) [inline site/content/contribute/gsoc/_index.md:47]:
```suggestion
All ideas in the *Core features* category propose direct changes to the Mattermost product, either within the server, or within the web client.
```

File: site/content/contribute/gsoc/_index.md [hunk 4]
```diff
@@ -110,7 +112,7 @@ This project is open to ideas that should be reflected in the student proposal.
 -   **Difficulty**: Easy
 -   **Mentors**: Daniel Espino García, Alejandro García Montoro
 
-### Add the concept of _Importance_ to messages
+#### Add the concept of _Importance_ to messages
```
Comment (justinegeffen) [inline site/content/contribute/gsoc/_index.md:115]:
```suggestion
#### Add the concept of *Importance* to messages
```

**Review by justinegeffen — APPROVED** [review]
Some really minor, non-blocking, nits, but otherwise this is great. :)

**Comment by agarciamontoro** [comment]
Thank you, everyone!

**Pull request merged**